testnets/shentutestnet/chain.json

```diff
@@ -0,0 +1,234 @@
+        },
+        "next_version_name": "v2.7.0",
+        "ibc": {
+          "type": "go",
+          "version": "v4.2.0"
+        }
+      },
+      {
+        "name": "v2.7.0",
+        "height": 12926000,
+        "proposal": 17,
+        "tag": "v2.7.0",
+        "ibc_go_version": "v4.2.1",
+        "ics_enabled": [
+          "ics20-1"
+        ],
+        "recommended_version": "v2.7.0",
+        "compatible_versions": [
+          "v2.7.0"
+        ],
+        "binaries": {
+          "linux/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.7.1/shentud_2.7.1_linux_amd64",
+          "darwin/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.7.1/shentud_2.7.1_arm64_macos",
+          "windows/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.7.1/shentud_2.7.1_win_x86_64.exe"
+        },
+        "next_version_name": "v2.7.2",
+        "ibc": {
+          "type": "go",
+          "version": "v4.2.1",
+          "ics_enabled": [
+            "ics20-1"
+          ]
+        }
+      },
+      {
+        "name": "v2.7.2",
+        "height": 14875800,
+        "proposal": 21,
+        "tag": "v2.7.2",
+        "ibc_go_version": "v4.2.1",
+        "ics_enabled": [
+          "ics20-1"
+        ],
+        "recommended_version": "v2.7.2",
+        "compatible_versions": [
+          "v2.7.2"
+        ],
+        "binaries": {
+          "linux/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.7.2/shentud_2.7.2_linux_amd64",
+          "darwin/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.7.2/shentud_2.7.2_arm64_macos",
+          "windows/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.7.2/shentud_2.7.2_win_x86_64.exe"
+        },
+        "next_version_name": "v2.8.0",
+        "ibc": {
+          "type": "go",
+          "version": "v4.2.1",
+          "ics_enabled": [
+            "ics20-1"
+          ]
+        }
+      },
+      {
+        "name": "v2.8.0",
+        "height": 15094300,
+        "proposal": 23,
+        "tag": "v2.8.0",
+        "ibc_go_version": "v4.2.1",
+        "ics_enabled": [
+          "ics20-1"
+        ],
+        "recommended_version": "v2.8.0",
+        "compatible_versions": [
+          "v2.8.0"
+        ],
+        "binaries": {
+          "linux/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.8.0/shentud_2.8.0_linux_amd64",
+          "darwin/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.8.0/shentud_2.8.0_arm64_macos",
+          "windows/amd64": "https://github.com/shentufoundation/shentu/releases/download/v2.8.0/shentud_2.8.0_win_x86_64.exe"
+        },
+        "next_version_name": "",
+        "ibc": {
+          "type": "go",
+          "version": "v4.2.1",
+          "ics_enabled": [
+            "ics20-1"
+          ]
+        }
+      }
+    ],
+    "ibc": {
+      "type": "go",
+      "version": "v4.2.1",
```

Comment:
This version number must match above (should be v7.4.0).
I think this is the last issue.

Reply:
Ah yes, forgot to update the testnet, sry ..